src/metadata-builder/RelationJoinColumnBuilder.ts

```diff
@@ -56,13 +56,14 @@ export class RelationJoinColumnBuilder {
      */
     build(joinColumns: JoinColumnMetadataArgs[], relation: RelationMetadata): {
       foreignKey: ForeignKeyMetadata|undefined,
+      columns: ColumnMetadata[],
```

Comment:
Maybe just rename this field to `joinColumns`?

Reply:
I thought about it - and I'm worried that it might be confusing, as this function already takes `joinColumns` (`JoinColumnMetadataArgs` type)` as argument, but in the result columns type is `ColumnMetadata[]` - so I've decided to follow naming same naming from types. Please let me know, if you have another idea how to resolve that naming conflict